app/src/main/java/com/framgia/englishconversation/screen/conversationdetail/ConversationDetailViewModel.java

```diff
@@ -80,10 +83,16 @@ public void onItemUserNameClick(TimelineModel item) {
                     mDetailActivity.startActivity(ProfileUserActivity.getInstance(mDetailActivity,
                             item.getCreatedUser()));
                 }
+
+                @Override
+                public boolean onItemLongClick(View viewGroup, TimelineModel item) {
+return false;
```

Comment:
ERROR: 'method def' child have incorrect indentation level 0, expected level should be one of the following: 12, 16.